CybercentreCanada/superset
# added option to emit a filter from a default column for a row in ag grid

### SUMMARY

Ability to specify a default column to emit in ag grid viz, so that when right clicking on any cell the filter emits from the same standard column(s)

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

chart creator
![image](https://user-images.githubusercontent.com/102618419/196709551-78039997-9830-4441-909b-20875459ea5e.png)
chart context menu
![image](https://user-images.githubusercontent.com/102618419/196709727-69de6c14-82f5-49f3-97c6-ec3878c56fd0.png)

### TESTING INSTRUCTIONS

- Create a chart
- enable emit filters
- add default column(s)
- In the dashboard view, when right clicking on that chart the option of emit default filters will appear in the drop down. 
- clicking emit default filters will emit the filter for the selected row, but with the specified default column
### ADDITIONAL INFORMATION
